csrankings.csv

```diff
@@ -12396,7 +12399,6 @@ S. C. Cheung,HKUST,https://www.cse.ust.hk/~scc,5RIgb3wAAAAJ
 S. C. Mukhopadhyay,Massey University,http://www.massey.ac.nz/massey/expertise/profile.cfm?stref=265530,fsu2yL8AAAAJ
 S. Dov Gordon,George Mason University,http://cs.gmu.edu/~gordon,NOSCHOLARPAGE
 S. E. M. O'Keefe,University of York,https://www-users.cs.york.ac.uk/~sok,QL1JHlUAAAAJ
-S. Hamed Hassani,University of Pennsylvania,https://www.seas.upenn.edu/~hassani,NOSCHOLARPAGE

```

Comment:
Why the deletion?

Reply:
His actual DBLP entry is "Seyed Hamed Hassani", which is already in the database. This is a duplicate that as far as I can tell serves no purpose.

Reply:
If you look at his DBLP entry, you will see that this is an alias. All aliases are included (for simplicity) and generated automatically if missing.

https://dblp.org/pers/hd/h/Hassani:Seyed_Hamed

Reply:
Oh okay. Sorry about that. I was not aware that this was being added automatically (or purposely). I thought someone (or something) had made a mistake. I'll submit an updated PR.

Reply:
No problem - thanks.